README.md

```diff
@@ -12,6 +12,10 @@
 
 <h4 id="top" align="center">Formerly known as "Pulse"</h1>
 
+## Documentation
```

Comment:
It actually should be somewhere below Table Of contents, and probably after `About the project` section

Reply:
Changed